.gitignore

```diff
@@ -142,3 +142,8 @@ Desktop.ini
 # ESLint
 ######################
 .eslintcache
+
+######################
+# Exclusions
+######################
+!lib/*.jar
```

Comment:
Do we still need it? If needed, plz add an empty line at the end.

Reply:
removed